src/Storages/MergeTree/MergeProgress.h

```diff
@@ -90,6 +89,25 @@ class MergeProgressCallback
                 std::memory_order_relaxed);
         }
     }
+
+private:
+    MergeListElement * merge_list_element_ptr;
+    UInt64 & watch_prev_elapsed;
+    MergeStageProgress & stage;
+    CancellationChecker cancellation_checker;
+
+    void updateWatch()
+    {
+        UInt64 watch_curr_elapsed = merge_list_element_ptr->watch.elapsed();
+        watch_prev_elapsed = watch_curr_elapsed;
+    }
+
+    void updateProfileEvents(const Progress & value, ProfileEvents::Event rows_event, ProfileEvents::Event bytes_event) const
```

Comment:
This method is not from original PR, is it used anywhere?

Reply:
Done, thanks